plugins/cairo-lang-macro/src/types/conversion.rs

```diff
@@ -109,11 +101,8 @@ impl TextSpan {
     }
 
     #[doc(hidden)]
-    pub fn from_owned_stable(span: StableTextSpan) -> Self {
-        Self {
-            start: span.start,
-            end: span.end,
-        }
+    pub fn free_owned_stable(span: StableTextSpan) {
+        let _ = span;
     }
```

Comment:
Shouldn't this be in the StableTextSpan's impl?

Reply:
We implement all conversions in `cairo-lang-macro` not `cairo-lang-macro-stable`,  so the latter only defines the binary representation that is passed through the ffi barrier (so it changes as rarely as possible).